src/app/providers/SignUp.js

```diff
@@ -0,0 +1,86 @@
+'use client'
+
+import Link from "@/app/providers/components/Link";
+import { Button, Checkbox, Container, TextField } from "@mui/material";
+import Box from "@mui/material/Box";
+import Divider from "@mui/material/Divider";
+import Grid from "@mui/material/Grid";
+import Typography from "@mui/material/Typography";
+import { useTheme } from "@mui/material/styles";
+import Image from "next/image";
+
+function SignUpForm() {
```

Comment:
It is a lot of code duplication here with an existing SignUpForm. I'd rather extract it to a component and pass exact text for header, subheader as props.